### DISCUSSION TITLE
Sky View Factor Calculator: dsm contains building, terrain and vegetation height

### DISCUSSION THREAD

**Discussion by fadelkoto** [UMEP-dev/UMEP#322] [Q&A]
Hello,

I am calculating the sky view factor with UMEP and I put the DSM of building, terrain, and vegetation as input in the filed "Building and ground DSM". Would the result of the caluclation be okay with this input instead of using the default input of building and ground only?

Thanks.

**Comment by biglimp** [comment]
Yes, the result will be okay but there are two things you should consider regarding your output SVF:

1.  svf-values are calculated ON TOP of vegetation and not underneath
2. Transmission of light through vegetation is not present. Trees act as a buildings
